pkg/chartmuseum/server/multitenant/server.go

```diff
@@ -49,7 +49,7 @@ type (
 		StorageBackend         cm_storage.Backend
 		TimestampTolerance     time.Duration
 		ExternalCacheStore     cache.Store
-		InternalCacheStore     map[string]*cacheEntry
+		InternalCacheStore     sync.Map
```

Comment:
Can we wrap the `sync.Map` inside a specialized structure , for example `type memoryCacheStore struct { sync.Map }` maybe ,  and wrap the `Load` and `Store` func ?  Doing a type assertion every where  seems ugly though .